roles/grafana_cloud_operator/tasks/grafana_oncall_standalone.yml

```diff
@@ -103,6 +103,28 @@
         alertmanager.yaml: "{{ encoded_alertmanager_secret_content }}"
   when: not integration_exists_for_cluster
 
+- name: Adds prometheus rule for cluster
+  kubernetes.core.k8s:
+    state: present
+    namespace: "openshift-monitoring"
+    definition:
+      apiVersion: monitoring.coreos.com/v1
+      kind: PrometheusRule
+      metadata:
+        name: "heartbeat"
```

Comment:
Maybe we can be more specific in the name so as to avoid any conflict. 

heartbeat => heartbeat-grafana-oncall